lib/jxl/modular/transform/enc_palette.cc

```diff
@@ -365,6 +445,14 @@ Status FwdPalette(Image &input, uint32_t begin_c, uint32_t end_c,
         }
         index = best_index;
         delta_used |= best_is_delta;
+        // Store delta for delta k-means.
```

Comment:
I assume that this is what we'll do later but not what we are doing now, is that correct?

Reply:
Yes.